### DISCUSSION TITLE
Support for adding alt text for image uploads

### DISCUSSION THREAD

**Discussion by GabrielSieben** [filamentphp/filament#8455] [Ideas]
Hello,

For websites which require ADA compliance (and, in the US, lawsuits have been won for websites that are not WCAG-compliant, _Robles v Dominos_), image uploads in the RichEditor and File Upload field should have the option to easily show a prompt to manually enter and render Alt text for images.

**Comment by danharrin** [comment]
The rich editor now has this functionality available in v4.

For the FileUpload, I would suggest adding a TextInput aside it to store the alt text.
